theano/tensor/nnet/abstract_conv.py

```diff
@@ -1396,7 +1417,7 @@ class BaseAbstractConv(Op):
 
     def __init__(self, convdim,
```

Comment:
In the \__props__ attribute above, you should add num_groups. See http://deeplearning.net/software/theano/extending/op.html#__props__ for an explanation. If that isn't clear, tell me!